prime-router/src/test/kotlin/cli/helpers/HL7DiffHelperTests.kt

```diff
@@ -93,6 +93,59 @@ class HL7DiffHelperTests {
         "2.16.840.1.114222.4.3.3.2.1.2&ISO||119365002^Specimen from wound^SCT^WND^Wound^L^0912017^Adobe_Code" +
         "^Wound||||56459004^Foot^SCT^FOT^Foot^L^09012017^Adobe_Code^Foot||||||Isolate,|||20230322|20230421124150+0000\n"
 
+    private val originalHL7Msg = """MSH|^~\&#|ProPhase^2.16.840.1.114222.4.1.238646^ISO|ProPhase^33D2215033^CLIA|CDC Prime^2.16.840.1.114222.4.1.237821^ISO|CDC Prime^2.16.840.1.114222.4.1.237821^ISO|20230816123358-0500||ORU^R01^ORU_R01|20230816123358|P|2.5.1|||||||||PHLabReport-NoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO|
```

Comment:
(nit, optional) It might be nice to clean this section up a bit. We probably can adjust the tests so that they all work off of the same baseline message, instead of having separate `originalMessage` and `originalHL7Msg`.

Reply:
Now, only keep 2 minimum in / out messages for new code coverage...